typescript/commands/quick_info.py

```diff
@@ -31,11 +31,14 @@ class TypescriptQuickInfoDoc(TypeScriptBaseTextCommand):
     re-runs quick info in case info has changed
     """
 
+    display_point = None
```

Comment:
Having this state out here, and setting it in the run method, then using it in a later method, is kinda ugly. Is there no way to pass it directly?